onnx/defs/math/defs.cc

```diff
@@ -1753,25 +2432,55 @@ ONNX_OPERATOR_SET_SCHEMA(
     10,
     OpSchema()
         .SetDoc(MatMulInteger_ver10_doc)
```

Comment:
Let's make this op entirely non-differentiable because all its inputs and outputs are integer tensors.

Reply:
They are integer, but that is just a compact representation, I think they should still be differentiable. Even double precision numbers do not provide completely continuous range of values. If we make those non-differentiable, then quantized training won't be possible.